samples/balloon/balloon.py

```diff
@@ -300,6 +300,10 @@ def detect_and_color_splash(model, image_path=None, video_path=None):
     parser.add_argument('--video', required=False,
                         metavar="path or URL to video",
                         help='Video to apply the color splash effect on')
+    parser.add_argument('--backbone', required=False,
+                        default="resnet50",
+                        metavar="<backbone>",
+                        help='Feature Pyramid Network backbone type')
```

Comment:
Selecting a backbone should be done in the Config class like the other network architecture details.

Reply:
Sure I can remove this. Just thought that a command line argument would provide convenience